packages/karma-runner/test/helpers/setupTests.ts

```diff
@@ -0,0 +1,24 @@
+import 'source-map-support/register';
```

Comment:
Is this file still used? I don't think the helper files are called anymore from the package.json

Reply:
Helper files configured using the `.mocharc.jsonc` file. I'm migrating mocha as I go along to update modules.